test/jdk/javax/net/ssl/DTLS/DTLSWontNegotiateV10.java

```diff
@@ -48,6 +48,8 @@ public class DTLSWontNegotiateV10 {
     private static final String DTLS = "DTLS";
     private static final String DTLSV_1_2 = "DTLSv1.2";
 
+    private static final int READ_TIMEOUT_SECS = Integer.getInteger("readtimeout", 30);
```

Comment:
Nit on line 37 (since I can't flag that directly): Maybe add 8310070 to the @bug line.

Reply:
Works for me.  Then as far as I'm concerned you're good to go.